components/alert/AlertMap.vue

```diff
@@ -31,29 +136,77 @@ export default Vue.extend({
       updateRate: 10000,
       updateCountdown: 0,
       updateCountdownInterval: undefined as undefined | number,
+      lastUpdated: new Date(0),
       interval: undefined as undefined | number,
+      zoom: 2,
+      prevZoom: 2,
+      center: [-128, 128],
+      url: "https://assets.ps2alerts.com/zones/" + zoneNameFilter(this.alert.zone).toLowerCase() + "/{z}/tile_{x}_{y}.png",
+      minZoom: 2,
+      maxZoom: 7,
+      zoomSnap: 1,
+      zoomDelta: 1,
+      bounds: [[0, 0], [-256, 256]],
+      maxBounds: [[0, 0], [-250, 250]],
+      viscosity: 0.1,
+      noWrap: true,
+      mapRegions: new Map<number, MapRegion>(),
+      map: {} as L.Map,
+      remaining: {} as LControl,
+      crs: this.$L.CRS.Simple, //this.$L.extend({}, this.$L.CRS.Simple, {wrapLng: [0, 256]}),
+      polys: this.$L.featureGroup([], {
+        pane: "hexPane"
+      }),
+      links: this.$L.featureGroup([], {
+        pane: "linkPane"
+      }),
+      linkStamps: {} as Record<string, number>,
+      badges: this.$L.featureGroup([], {
+        pane: "badgePane"
+      }),
+      textBadges: this.$L.featureGroup([], {
+        pane: "badgeTextPane"
+      }),
+      currentIndex: -1,
+      captureIndices: [] as number[],
+      tickLabels: [] as string[],
+      oldSliderVal: 0,
+      sliderVal: 0,
+      sliderMax: 0,
+      historyCache: [] as InstanceFacilityControlEntriesResponseInterface[],
+      outfitData: new Map<string, InstanceOutfitAggregateResponseInterface>(),
+      zoomInSound: new Audio('/audio/UI_INTERFACE_MAP_ZOOM_IN.wav'),
+      zoomOutSound: new Audio('/audio/UI_INTERFACE_MAP_ZOOM_OUT.wav'),
       // data: {} as InstanceFactionCombatAggregateResponseInterface,
     }
   },
   computed: {
-    // updateCountdownPercent(): number {
-    //   return (100 / (this.updateRate / 1000)) * this.updateCountdown
-    // },
+    updateCountdownPercent(): number {
+      return (100 / (this.updateRate / 1000)) * this.updateCountdown
+    },
   },
   watch: {
-    // 'alert.state'() {
-    //   if (this.alert.state === Ps2alertsEventState.ENDED) {
-    //     this.clearTimers()
-    //     this.pull()
-    //   }
-    // },
+    'alert.state'() {
+      if (this.alert.state === Ps2alertsEventState.ENDED) {
+        this.clearTimers()
+        this.pull()
+        this.remaining.$el.remove();
+      }
+    },
   },
   beforeDestroy() {
     this.reset()
   },
   created() {
     this.reset()
-    this.init()
+  },
+  mounted(){
+    this.$nextTick(() => {
```

Comment:
I'm referencing the map DOM element and placing the code in mounted would try to reference the map element before it actually was created, the solution I found online was to wait a tick so it would be created